posthog/management/commands/update_batch_export_schedules.py

```diff
@@ -11,29 +15,61 @@ class Command(BaseCommand):
+        parser.add_argument("--batch-export-id", default=None, type=str, help="Single batch export ID to update")
+        parser.add_argument("--team-id", default=None, type=str, help="Team ID for which to update all batch exports")
+        parser.add_argument(
+            "--destination-type", default=None, type=str, help="Update all batch exports for a given destination type"
+        )
+
+    def _update_batch_export_schedule(self, batch_export: BatchExport):
+        logger.info("Updating batch export schedule...", batch_export_id=str(batch_export.id))
+
+        try:
+            sync_batch_export(batch_export, created=False)
+        except Exception:
+            logger.exception("Error updating batch export schedule", batch_export_id=str(batch_export.id))
 
     def handle(self, **options):
+        logger.setLevel(logging.INFO)
         batch_export_id = options["batch_export_id"]
-        # verbose = options["verbosity"] > 1
 
-        if not batch_export_id:
-            raise CommandError("Batch export ID is required")
+        batch_exports: list[BatchExport] = []
 
-        # TODO - run in a loop once we support multiple batch exports
-        try:
-            batch_export: BatchExport = BatchExport.objects.get(id=batch_export_id, deleted=False)
-        except BatchExport.DoesNotExist:
-            raise CommandError("Batch export not found")
-        except BatchExport.MultipleObjectsReturned:
-            raise CommandError("More than one existing batch export found (this should never happen)!")
+        if batch_export_id:
+            try:
+                batch_export: BatchExport = BatchExport.objects.get(id=batch_export_id, deleted=False)
+                batch_exports.append(batch_export)
+            except BatchExport.DoesNotExist:
+                raise CommandError("Batch export not found")
+            except BatchExport.MultipleObjectsReturned:
+                raise CommandError("More than one existing batch export found (this should never happen)!")
+        elif options["destination_type"] or options["team_id"]:
+            query_set = BatchExport.objects.filter(deleted=False)
 
-        logger.info("Updating batch export schedule...", batch_export_id=batch_export.id)
+            if options["destination_type"]:
+                query_set = query_set.filter(destination__type=options["destination_type"])
 
-        try:
-            sync_batch_export(batch_export, created=False)
-        except Exception:
-            logger.exception("Error updating batch export schedule", batch_export_id=batch_export.id)
+            if options["team_id"]:
+                query_set = query_set.filter(team__id=int(options["team_id"]))
+
+            batch_exports = list(query_set)
+        else:
+            # Do we want to allow it to run on all batch exports?
+            raise CommandError("No batch export ID or destination type or team ID provided")
```

Comment:
I think it's fine to not allow this for now, just in case. Eventually, though, I am not against it.